roles/os_update_latest/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+oul_package_names: "*"
+oul_exclude: 'atomic-openshift* tuned-profiles-atomic-openshift*'
```

Comment:
This needs to be updated to differentiate between `openshift_deployment_type == 'origin' and 'openshift-enterprise'` as the package names are `*origin*` and `*atomic-openshift*` respectively.